lib/absinthe/phase/document/execution/resolution.ex

```diff
@@ -143,16 +143,13 @@ defmodule Absinthe.Phase.Document.Execution.Resolution do
     |> to_result(bp_field, full_type)
     |> walk_result(acc, bp_field, full_type, info)
   end
+  defp build_result({:error, params}, acc, bp_field, info, _)
+   when is_list(params) or is_map(params) do
```

Comment:
Just to match with the existing style in Absinthe, please stick the guard on the same line as the function definition (yeah, I know it makes for a long line).